fe/fe-core/src/main/java/org/apache/doris/task/AgentBatchTask.java

```diff
@@ -50,23 +51,35 @@
 import com.google.common.collect.Lists;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.apache.thrift.TException;
 
 import java.util.HashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.function.BiFunction;
 
 /*
  * This class group tasks by backend
  */
 public class AgentBatchTask implements Runnable {
     private static final Logger LOG = LogManager.getLogger(AgentBatchTask.class);
 
+    /**
+     * for tracing this batch task
+     */
+    private final long batchId;
+
     // backendId -> AgentTask List
-    private Map<Long, List<AgentTask>> backendIdToTasks;
+    private final Map<Long, List<AgentTask>> backendIdToTasks;
 
     public AgentBatchTask() {
-        this.backendIdToTasks = new HashMap<Long, List<AgentTask>>();
+        if (FeConstants.runningUnitTest) {
+            this.batchId = 1L;
```

Comment:
I wouldn't recommend it if it's just for UT.
This can be done by way of mocks without intruding into the code.

Reply:
Actually BackupJobTest.java encountered exception without this way, it seems to be caused by calling `Env.getCurrentEnv().getNextId();` in ut environment, where getCurrentEnv() seems to return null. Is there a way can resolve the problem during mocks?

Reply:
I 've found out a better way to do the corresponding mock